Client/Frontend/Browser/Search/SearchEngines.swift

```diff
@@ -15,6 +15,12 @@ private let ShowSearchSuggestionsOptIn = "search.suggestions.showOptIn"
 private let ShowSearchSuggestions = "search.suggestions.show"
 private let customSearchEnginesFileName = "customEngines.plist"
 
+// BRAVE TODO: Move to newer Preferences class.
```

Comment:
Can you please make a ticket for this, put it under the Preferences epic and then update this TODO to include the ticket number.